.github/workflows/release-new-charts.yaml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - arh-dspdc-1344-add-load-tag
```

Comment:
I've added this branch here to test this PR prior to merge, this will be removed prior to merge.